### PR TITLE
Design review

### PR DESCRIPTION
**Pull request by beamalsky**
## Overview

Connects #136. @jeancochrane this is an initial round of design fixes ready for your eyes. I've branched one bigger chunk of work, redoing the Fobi styles to match the rest of the app, into #141.

## To do

- [x] Clean up and reorganize `custom.css` 
- [x] Get sidebar working properly on mobile/small windows
- [x] Standardize form formatting outside of Fobi (including authentication templates!)
- [x] Add explainer text to PLDP creation forms
- [x] Reconcile styling of Fobi views with rest of app - moved to #141 
- [x] Style publish template
- [x] Make sure small windows don't break Fobi views - moved to #141 
- [x] Consider making the sidebar [sticky](https://getbootstrap.com/docs/4.1/examples/dashboard/) - experimented with this and decided to keep as-is
- [x] Make tables reasonably responsive

### CODE REVIEW

File: surveys/static/css/custom.css [hunk 1]
```diff
@@ -1,29 +1,36 @@
...
 a:focus {
     color: var(--orange-color);
-    font-weight: 900;
-    background: var(--off-white-color);
-    border-radius: 5px;
-    padding: 3px 7px;
+    font-weight: 800;
+    /* background: var(--off-white-color); */
+    /* border-radius: 5px; */
+    /* padding: 3px 7px; */
```
Comment (jeancochrane) [inline surveys/static/css/custom.css]: A few comments to delete here.

File: surveys/static/css/custom.css [hunk 2]
```diff
@@ -72,13 +79,9 @@ span {
...
     max-width: 250px;
-    background: var(--dark-green-color);
-    color: var(--darker-green-color);
-    transition: all 0.3s;
-}
-
-.sidebar h3 {
-    color: var(--darker-green-color);
+    background: var(--light-green-color);
+    transition: all 0.2s;
```
Comment (jeancochrane) [inline surveys/static/css/custom.css]: These animations are cute!

File: surveys/templates/partials/sidebar.html [hunk 3]
```diff
@@ -68,15 +71,29 @@ <h3><b>Just Spaces</b></h3>
...
+            $sidebar.toggleClass('active');
         });
+      });
+
+      $(window).resize(function() {
+        resizeSidebar();
+      });
+
+      function resizeSidebar() {
+        if (window.innerWidth <= 1000) {
```
Comment (jeancochrane) [inline surveys/templates/partials/sidebar.html]: What's your reasoning behind 1000px as the sidebar breakpoint? Bootstrap 4 uses 992px as the breakpoint for "large screens", that would make sense to me to use here as well: https://getbootstrap.com/docs/4.0/layout/overview/#responsive-breakpoints

File: surveys/templates/registration/login.html [hunk 4]
```diff
@@ -19,22 +22,14 @@
...
-    </tr>
-    <tr>
-        <td>{{ form.password.label_tag }}</td>
-        <td>{{ form.password }}</td>
-    </tr>
-    </table>
 
-    <input type="submit" value="login" class="btn btn-success float-right">
+    {{form.username | as_crispy_field}}
+    {{form.password | as_crispy_field}}
```
Comment (jeancochrane) [inline surveys/templates/registration/login.html]: These inputs are a bit too wide for my taste on large screens:

Screenshot from 2019-05-22 10-18-09

File: surveys/static/css/custom.css [hunk 5]
```diff
@@ -144,25 +125,38 @@ span {
...
-    color: var(--dark-green-color);
-    background: var(--darker-green-color);
+    padding: 5px 25px;
+}
+
+.sidebar-blurb a {
+  background: var(--off-white-color);
+  border-radius: 5px;
+  padding: 3px 7px;
+}
```
Comment (jeancochrane) [inline surveys/static/css/custom.css]: I like this styling for the login/logout buttons, but when the links (to DataMade and UCD) have the same styling it makes it a little bit hard to figure out what's an action button and what's a link:

Screenshot from 2019-05-21 18-25-40

It'd be nice to mute the styles of the links a bit -- they're great to have, but they're not as semantically important in this context.

File: surveys/static/css/custom.css (hunk 2 above)
Comment (beamalsky) [reply surveys/static/css/custom.css]: :)

File: surveys/templates/registration/login.html (hunk 4 above)
Comment (beamalsky) [reply surveys/templates/registration/login.html]: Good point!

File: surveys/templates/partials/sidebar.html (hunk 3 above)
Comment (beamalsky) [reply surveys/templates/partials/sidebar.html]: Just that it felt right! 992px is more sensible.

**Pull request merged**